src/components v2/Atoms/Components/VSelect/InputSearch/InputSearch.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="ph-autocomplete-search">
+    <span v-if="prefixIcon" class="ph-icon-wrapper ph-autocomplete-prefix-icon">
+      <p-icon ref="prefixIcon" :name="prefixIcon" type="med" />
+    </span>
+    <input
+      v-show="hideInputOnSelected"
+      data-testid="autocomplete-input"
+      class="vs__search"
+      :disabled="disabled"
+      :class="[
+        prefixIcon && 'ph-autocomplete-search-indent',
+        disabled && 'ph-autocomplete-disabled',
+      ]"
+      v-bind="attributes"
+      v-on="events"

```

Comment:
**Issue:** we should use explicit bindings. 

We don't know what is going to happen here. Define props explicitly in the props section and bind all relevant emits. That way we know what it can and cant do.